examples/plot_bands_functionality.py

```diff
@@ -0,0 +1,107 @@
+"""
+Plot Bands of Satellite Imagery with EarthPy
```

Comment:
Plot raster data layers using earthpy 

^^ something like that might be better. it's more generic